Bento/Renderable/AnyRenderable.swift

```diff
@@ -99,13 +106,16 @@ private class AnyRenderableBox<Base: Renderable>: AnyRenderableBoxBase where Bas
     }
 }
 
-private class AnyRenderableBoxBase {
+class AnyRenderableBoxBase {
     var reuseIdentifier: String { fatalError() }
 
     var viewType: Any.Type { fatalError() }
 
     init() {}
 
+    func asAnyRenderable() -> AnyRenderable {
```

Comment:
there doesn't seem any strong reason of not using the initialiser directly.

Reply:
This is just to enable chaining calls 
```swift
DeletableComponent(
	            source: self,
	            deleteActionText: deleteActionText,
	            backgroundColor: backgroundColor,
	            didDelete: didDelete
	        )
.asAnyRenderable()
```